src/internal_events/file.rs

```diff
@@ -261,6 +261,29 @@ mod source {
         }
     }
 
+    #[derive(Debug)]
+    pub struct PathGlobbingFailed<'a> {
+        pub path: &'a Path,
+        pub error: &'a Error,
+    }
+
+    impl<'a> InternalEvent for PathGlobbingFailed<'a> {
+        fn emit_logs(&self) {
+            error!(
+                message = "Failed globbing path.",
+                path = ?self.path,
+                error = ?self.error,
```

Comment:
Should we be using the `Display` trait here and for `path` per https://github.com/timberio/vector/blob/master/CONTRIBUTING.md#logging-style? That is: `path = %self.path, error = %self.error`.

Reply:
Yep, I thought `Path` only had `Debug` but the compiler was smarter and suggested `.display()` method.

Reply:
I'll also update the rest of the events in the file to follow this.